config/lit_module/jit_models/jit.yaml

```diff
@@ -0,0 +1,9 @@
+_target_: ahcore.lit_module.AhCoreLightningModule
+
+model:
+  _target_: ahcore.models.jit_model.AhcoreJitModel.from_jit_path
+  jit_path: ???
```

Comment:
Since ckpt_path is also standardly null, I think jit_path should be as well. We check in the inference entrypoint whether the values are correctly set, so the ??? is not necessary.

Reply:
This is done.